solutions/devsprint-julio-fernandes-2/FinanceAppTests/UserProfileTests.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  UserProfileTests.swift
+//  FinanceAppTests
+//
+//  Created by Pedro Henrique Martins Barbosa on 20/04/22.
+//
+
+import XCTest
+@testable import FinanceApp
+
+final class UserProfileTests: XCTestCase {
+    var sut: UserProfile?
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        
+        sut = UserProfile(
+            name: "Paulo Henrique Silva",
+            phone: "(81) 998904067",
+            email: "psilva@gmail.com",
+            address: "Rua São João",
+            account: Account(agency: "9345", account: "0023519034")
+        )
+    }
+    
+    override func tearDownWithError() throws {
+        sut = nil
+        try super.tearDownWithError()
+    }
+    
+    func test_formattedAgency_filledAgency_shouldReturnValidFormat() {
+        let formattedAgency: String = "Agency: 9345"
+        if let value = sut {
+            XCTAssertEqual(formattedAgency, value.account.formattedAgency())
+        }
+    }
+    
+    func test_formattedAgency_filledAgency_shouldHaveAgencyPrefix() {
+        let formattedAgency: String = "Agency"
+        if let value = sut {
```

Comment:
A forma como você tratou o sut criando um if let não esta errado, porem não temos o retorno se ele for nulo. 

O que acha então de fazer assim? 

let sut = try XCTUnwrap(sut) 

E no metodo você adiciona um throws